var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -414,9 +415,17 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     # and https://github.com/abseil/abseil-cpp/issues/1665
     patch("absl_neon.patch", when="@2.16.1: target=aarch64:")
 
-    patch("Find_ROCm_Components_Individiually.2.16.patch", when="@2.16-rocm-enhanced +rocm")
+    patch(
+        "https://github.com/ROCm/tensorflow-upstream/commit/c467913bf4411ce2681391f37a9adf6031d23c2c.patch?full_index=1",
+        sha256="82554a84d19d99180a6bec274c6106dd217361e809b446e2e4bc4b6b979bdf7a",
+        when="@2.16-rocm-enhanced +rocm",
+    )
     patch("Find_ROCm_Components_Individiually.patch", when="@2.14-rocm-enhanced +rocm")
-    patch("set_jit_trueLT_false.patch", when="@2.14-rocm-enhanced: +rocm")
```

Comment:
Did you intend to also remove this patch?

Reply:
Yes I intend to remove that patch as well. That change was pushed here:
https://github.com/ROCm/tensorflow-upstream/commit/f4f4e8698b90755b0b5ea2d9da1933b0b988b111

Reply:
Should've been clearer, did you intend to *also* remove the patch file from the Spack repository?

Reply:
Ya I did, thanks for catching that